### DISCUSSION TITLE
Welcome to instagram-dark-extension Discussions!

### DISCUSSION THREAD

**Discussion by antoinekm** [onRuntime/instagram-dark-extension#40] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by qub2qub** [comment]
Hi, you've made a really cool theme, completely black - I love it, 
but it has disappeared from chrome extensions store, 
how can I add it to my chrome browser now?
